perfTest/init.js

```diff
@@ -0,0 +1,13 @@
+const { Pool } = require("pg");
```

Comment:
Moved this into a js file so that it could run on a remote db, without requiring psql